sui-execution/latest/sui-adapter/Cargo.toml

```diff
@@ -17,20 +17,29 @@ serde.workspace = true
 move-binary-format.workspace = true
 move-bytecode-utils.workspace = true
 move-core-types.workspace = true
-move-vm-config.workspace = true
-move-vm-types.workspace = true
+move-vm-config = { workspace = true, features = ["gas-profiler"] }
+move-vm-types = { workspace = true, features = ["gas-profiler"] }
```

Comment:
Similar question about feature enablement and whether it's needed here and below.